test/test-network/conf/dhcp-client-timezone-router.network

```diff
@@ -4,5 +4,7 @@ Name=veth99
 [Network]
 IPv6AcceptRA=false
 DHCP=ipv4
+
+[DHCP]
```

Comment:
I find it weird that the test passes with or without this line :-) It seems to me that this section isn't covered by the test at all.

Reply:
Is it? Without this, at least on fedora 28, the tests using this this conf file logs something in the journal.

Reply:
But none of the tests fails as far as know. Nobody looks at the journal if nothing fails :-)

Reply:
I guess the same goes for the issue fixed in https://github.com/systemd/systemd/pull/10618/commits/f7601e3a4fb4a611b60d4fe997ab1162941ccc8f. It's true that `networkd` complains loudly:
```sh
/run/systemd/network/25-geneve.netdev:10: Unknown lvalue 'UDP6ZeroChecksumTx' in section 'GENEVE'
/run/systemd/network/25-geneve.netdev:11: Unknown lvalue 'UDP6ZeroChecksumRx' in section 'GENEVE'
/run/systemd/network/25-geneve.netdev:10: Unknown lvalue 'UDP6ZeroChecksumTx' in section 'GENEVE'
/run/systemd/network/25-geneve.netdev:11: Unknown lvalue 'UDP6ZeroChecksumRx' in section 'GENEVE'
```
but the test passes no matter what is used in the config.

Reply:
Anyway, strictly speaking, that has nothing to do with the PR and can be safely ignored :-)